packages/cdai/src/components/IdeCard/IdeCard.js

```diff
@@ -294,6 +294,7 @@ export class IdeCard extends React.Component {
             <div
               className={`${idePrefix}-card--tile-img`}
               {...idAttribute(`ide-card-${cardId}-launch-Image`)}
+              alt="Opens in new tab"
```

Comment:
we shouldn't have any hard coded strings for translation purposes. this really should be sent as a prop.

Reply:
@davidmenendez Makes sense. I have updated the component to have a new optional string prop `externalLinkText`, which is passed down through `renderCardContent` and `renderCardHeader` to this point. Currently I'm retaining the English string "Opens in new tab" as the *default* value of this prop, as I see is done in a few other places in this codebase.